bot.py

```diff
@@ -98,6 +99,26 @@ def get_members_by_role(role):
     if message.author == client.user:
         return
     member = server.get_member_named(str(message.author))
+    
+    #Post a message as WaifuBot
+    if "Love,\nWaifuBot" in message.content:
+        if message.channel.is_private:
+            return
+        authorized_users = [
```

Comment:
This is fine for now, but next version should just check if they are in the admin group.